app/controllers/state_file/questions/terms_and_conditions_controller.rb

```diff
@@ -2,7 +2,10 @@ module StateFile
   module Questions
     class TermsAndConditionsController < QuestionsController
       def edit
-        @li_items = I18n.t('state_file.questions.terms_and_conditions.edit.list_items_html', privacy_policy_link: state_file_privacy_policy_path).split('</li>')
+        ownership = current_state_code == "nj" ?
+          "#{I18n.t('general.code_for_america')} #{I18n.t('general.and')} #{I18n.t('general.nj_ooi')}" :
+          I18n.t('general.code_for_america')
+        @li_items = I18n.t('state_file.questions.terms_and_conditions.edit.list_items_html', ownership: ownership, privacy_policy_link: state_file_privacy_policy_path).split('</li>')
```

Comment:
gotta admit i don't quite understand how it works that these are all missing a closing tag and yet it renders fine?

Reply:
great point -- the HTML is imperfect (all missing `</li>` because of the `split()` as you noted) but the browser is forgiving and knows how to render it anyway. but I'll do some timeboxed work to see if I can easily get it to proper HTML...

Reply:
actually @jenny-heath  - the only reason any of these shenanigans with `split()` and the missing `</li>` exist is to support the feature flag just below, which was introduced 12 months ago. Any idea if we can drop support for one of the cases of that flag now that a long time has passed? Then we wouldn't need any of this workaround code at all!

Happy to make that a follow-up ticket too, if that requires a larger discussion. But just wanted to check in case it's easy.

Reply:
Closing the loop - confirmed with Tiffany that both states of the flag are still needed. I switched the `split()` to a `scan()` with a regex so that the HTML is proper.